python/tests/examples/test_examples.py

```diff
@@ -46,6 +46,8 @@
             "workflows/searx_agent.py",
             # Requires BeeAI platform to be running
             "agents/experimental/remote.py",
+            # Requires Code Interpreter to be running
```

Comment:
we may need a condition here (or a soft fail) (check if port open?)
or split the tests that require backends into a different group

Reply:
just check for the env variable